### ISSUE TITLE
Add metadata getter for ChemRxiv

### ISSUE THREAD

**Issue by cthoyt** [manubot/manubot#283]
Along a similar vein as #282, Manubot does not seem to provide any ChemRxiv support. This is a bit more clear cut as a feature addition that I would be happy to send a PR for to allow either for `chemrxiv:` as a prefix, or to similarly to parse a ChemRxiv identifier out of a URL to a ChemRxiv work.

**Comment by cgreene** [comment]
Does chemRxiv provide a doi and can you just use the doi prefix?

**Comment by cthoyt** [comment]
> Does chemRxiv provide a doi and can you just use the doi prefix?

Yes, they issue DOIs following the pattern: `10.26434/CHEMRXIV.<chemrxiv id>`, so I suppose this could be trivially implemented with a rewrite rule rather than hitting their API, assuming all of the correct data ends up in Crossref (assuming that's the API you're using, I didn't check)

**Comment by dhimmel** [comment]
Currently, ChemRxiv works can be cited like:

- `@doi:10.26434/chemrxiv.13623173.v1` or `@10.26434/chemrxiv.13623173.v1`
- `@https://doi.org/10.26434/chemrxiv.13623173.v1`
- `@https://chemrxiv.org/articles/preprint/Proteomic_Database_Search_Engine_for_Two-Dimensional_Partial_Covariance_Mass_Spectrometry/13623173`

Citing `@chemrxiv:13623173.v1` would look cleaner, but on the other hand we want to keep the number of custom prefixes that we have to maintain low (and there are lot's of preprint servers and figshare clones). Let's keep an eye on #282. If chemRxiv is added to https://registry.identifiers.org/ that would be the preferred solution.

> assuming all of the correct data ends up in Crossref (assuming that's the API you're using, I didn't check)

For DOI citations like `@10.26434/chemrxiv.13623173.v1`, the metadata is coming from crossref via the crosscite/datacite DOI content negotiation endpoint:

https://github.com/manubot/manubot/blob/3ff3000f76dcf82a30694d076a4da95326e3f6ae/manubot/cite/doi.py#L125

Have you found that anything is wrong with getting the metadata from the DOI?
